psi4/driver/driver_cbs.py

```diff
@@ -985,65 +1292,146 @@ def _validate_cbs_inputs(cbs_metadata, molecule):
         # 1a) all items must have wfn
         if "wfn" not in item:
             raise ValidationError(f"Stage {iitem} doesn't have defined level of theory!")
-    # 1b) all items must have basis set
+        # 1b) all items must have basis set
         if "basis" not in item:
             raise ValidationError(f"Stage {iitem} doesn't have defined basis sets!")
-    # 2a) process required stage parameters and assign defaults
+        # 2) process required stage parameters and assign defaults
```

Comment:
Is there a specification for the metadata dicts anywhere? I have not been able to follow this code without one.